tokens/mnft.svg

```diff
@@ -0,0 +1,10 @@
5.792969 14.164062 25.761719 14.097656 25.726562 C 14.027344 25.691406 13.941406 25.664062 13.902344 25.664062 C 13.863281 25.664062 13.757812 25.636719 13.660156 25.605469 C 13.367188 25.503906 13.242188 25.488281 13.253906 25.558594 M 8.863281 25.597656 C 8.816406 25.6875 8.824219 25.695312 8.90625 25.695312 C 8.992188 25.695312 9.074219 25.621094 9.039062 25.570312 C 9.007812 25.515625 8.902344 25.53125 8.863281 25.597656 M 9.265625 25.617188 C 9.238281 25.726562 9.265625 25.761719 9.378906 25.761719 C 9.472656 25.761719 9.539062 25.648438 9.488281 25.570312 C 9.445312 25.496094 9.289062 25.53125 9.265625 25.617188 M 18.59375 25.566406 C 18.558594 25.589844 18.585938 25.597656 18.695312 25.597656 C 18.777344 25.601562 18.851562 25.585938 18.863281 25.566406 C 18.890625 25.527344 18.65625 25.527344 18.59375 25.566406 M 8.199219 25.664062 C 8.078125 25.714844 8.117188 25.761719 8.265625 25.742188 C 8.453125 25.714844 8.488281 25.703125 8.464844 25.664062 C 8.441406 25.625 8.296875 25.625 8.199219 25.664062 M 7.636719 25.761719 C 7.707031 25.902344 7.839844 25.875 7.792969 25.726562 C 7.777344 25.683594 7.742188 25.664062 7.679688 25.664062 C 7.589844 25.664062 7.589844 25.664062 7.636719 25.761719 M 18.3125 25.6875 C 18.34375 25.695312 18.394531 25.695312 18.425781 25.6875 C 18.453125 25.679688 18.429688 25.671875 18.367188 25.671875 C 18.304688 25.671875 18.28125 25.679688 18.3125 25.6875 M 14.414062 25.824219 C 14.425781 25.839844 14.476562 25.855469 14.53125 25.855469 C 14.582031 25.855469 14.625 25.839844 14.625 25.824219 C 14.625 25.804688 14.574219 25.792969 14.511719 25.792969 C 14.441406 25.792969 14.402344 25.804688 14.414062 25.824219 M 15.574219 25.941406 C 15.617188 25.949219 15.679688 25.949219 15.71875 25.941406 C 15.757812 25.933594 15.726562 25.929688 15.648438 25.929688 C 15.570312 25.929688 15.535156 25.933594 15.574219 25.941406 M 13.675781 28.589844 C 13.707031 28.707031 13.871094 28.863281 13.96875 28.863281 C 14.082031 28.863281 14.09375 28.8125 13.984375 28.769531 C 13.933594 28.746094 13.886719 28.710938 13.875 28.683594 C 13.867188 28.660156 13.828125 28.640625 13.792969 28.640625 C 13.757812 28.640625 13.726562 28.621094 13.726562 28.59375 C 13.726562 28.570312 13.710938 28.539062 13.691406 28.523438 C 13.664062 28.507812 13.660156 28.527344 13.675781 28.589844 M 15.027344 28.796875 C 14.996094 28.847656 15.054688 28.863281 15.15625 28.832031 C 15.207031 28.816406 15.222656 28.804688 15.195312 28.800781 C 15.164062 28.800781 15.121094 28.792969 15.09375 28.78125 C 15.066406 28.773438 15.039062 28.777344 15.027344 28.796875 M 18.113281 28.800781 C 18.074219 28.824219 18.078125 28.832031 18.136719 28.832031 C 18.175781 28.832031 18.207031 28.816406 18.207031 28.800781 C 18.207031 28.761719 18.175781 28.761719 18.113281 28.800781 M 17.03125 28.820312 C 17.0625 28.832031 17.113281 28.832031 17.144531 28.820312 C 17.175781 28.8125 17.148438 28.808594 17.089844 28.808594 C 17.027344 28.808594 17 28.8125 17.03125 28.820312 "/>
```

Comment:
Now it's 372 KB file! The initial reason was to decrease the size of the file, don't increase it. Please make sure it's less than 5-7 KB.